xbmc/video/VideoDatabase.cpp

```diff
@@ -6335,11 +6336,46 @@ void CVideoDatabase::UpdateTables(int iVersion)
                             "AND itemType = %i",
                             VideoAssetTypeOwner::USER, VideoAssetType::VERSION));
   }
+
+  if (iVersion < 131)
+  {
+    // Remove quality-like predefined version types
+
+    // Retrieve current utilization per type
+    m_pDS->query("SELECT vvt.id, vvt.name, count(vv.idType) "
+                 "FROM videoversiontype vvt "
+                 "  LEFT JOIN videoversion vv ON vvt.id = vv.idType "
+                 "WHERE vvt.id = 40405 OR vvt.id BETWEEN 40418 AND 40430 "
+                 "GROUP BY vvt.id, vvt.name");
```

Comment:
isn't `GROUP BY vvt.id` enough, or will the RDBMS complain otherwise?

Reply:
The query executes without, it's an old habbit, I had plenty of syntax errors when not including all non-aggregated fields in the group by. Must have been a slightly different situation.